core/jvm/src/test/scala/scalaz/zio/RetrySpec.scala

```diff
@@ -151,8 +177,30 @@ class RetrySpec(implicit ee: org.specs2.concurrent.ExecutionEnv) extends TestRun
     }
     val strategy = Schedule.spaced(200.millis).whileInput[String](_ == "KeepTryingError")
     val retried  = unsafeRun(retryCollect(io, strategy))
-    val expected = (Left("GiveUpError"), List(1, 2, 3, 4, 5).map((200.millis, _)))
-    retried must_=== expected
+    val expected =
+      (Left("GiveUpError"), List(1, 2, 3, 4, 5).map((200.millis.relative, _)))
+
+    val results1 = (
+      retried._1,
+      retried._2.map(
+        r1 =>
+          unsafeRun(for {
+            d1 <- r1._1.run
+          } yield d1)
+      )
+    )
+
+    val results2 = (
+      expected._1,
+      expected._2.map(
+        r1 =>
+          unsafeRun(for {
```

Comment:
Push this (and the other 2) unsafe runs to the top, by making a top-level for comprehension. Each test should have just 1 unsafeRun.